components/js/override.js

```diff
@@ -31,7 +31,7 @@ define(['angular'], function(angular) {
         // 'aboutURL': 'staticFeeds/about-frame.json',
         // 'aboutPageURL': 'staticFeeds/about-page.json',
         // 'bannersURL': 'staticFeeds/sample-banners.json',
-        // 'messagesURL': 'staticFeeds/sample-messages.json',
+        'messagesURL': 'staticFeeds/sample-messages.json',
```

Comment:
No, please. Let's keep the configuration to use the sample messages out of the shipping product, so that there's less chance a downstream application would inadvertently accept this as a default and ship sample messages to production.